jina/resources/helloworld.html

```diff
@@ -1 +1,28 @@
+            </a>
+        </div>
+        <div
+             class="about" style="max-width: 50%;padding: 10px;">
+            <h2 style="color: #1E6E73">What just happened?</h2>
+            <span>This is Jina's <pre style="display: inline;">hello-world</pre>,end-to-endly. It downloads Fashion-MNIST dataset and indexes 60,000 images via Jina search framework. The index is stored into multiple <i>shards</i>. We then randomly sample unseen images as <i>Queries</i>, ask Jina to retrieve relevant results. Below is Jina's retrievals, where the left-most column is query image.</span>
+            <br>
+            <span> Intrigued? Learn more about Jina and <a href="https://opensource.jina.ai">checkout our Github!</a></span>
```

Comment:
That should point to https://get.jina.ai/ since it is where the Readme is displayed.